### DISCUSSION TITLE
How to have a unique db index across two columns?

### DISCUSSION THREAD

**Discussion by glani** [keystonejs/keystone#7777] [Feature requests]
Is it possible to add somehow a custom definition to scheme.prisma

@@unique([firstName, lastName], name: "TheLastUser_firstName_lastName_unique_constraint")

to be able to control constraints on the db ?

**Comment by dcousens** [comment]
Hi @glani,
We are currently tracking this feature request in https://github.com/keystonejs/keystone/discussions/7705,  don't hesitate to comment on that discussion :+1:
